src/main/java/client/Client.java

```diff
@@ -0,0 +1,29 @@
+package main.java.client;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+
+public class Client {
+
+    private static final int PORT = 8080;
+    private static final String HOST = "127.0.0.1";
+
+
+    public static void main(String[] args) {
+        try {
+            Socket socket = new Socket(HOST, PORT);
+            BufferedReader br = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            String str = null;
+            while ((str = br.readLine()) != null ){
+                System.out.println(str);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=3&issues=AWQTV3ID_bEzQ-kUL94L&open=AWQTV3ID_bEzQ-kUL94L)

[ISSUE_KEY]: AWQTV3ID_bEzQ-kUL94L